### PR TITLE
Fix accessibility issues in Summary Table

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/19444

*Description of changes:*
- Errors with button menu closed (command-name / button-name) 
- Error on h4s for control descriptions
- AA issues associated with the UPopover (nested interactivity/hidden-focused / colour contrast) are ignored in the test.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-115-235w03tq.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-115-235w03tq.web.app

File: btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange.cy.ts [hunk 1]
```diff
@@ -95,21 +95,44 @@ describe('accessibility -> Beneficial Owner Change', () => {
     // cy.checkA11y('[data-cy=isUnableToObtainOrConfirmInformation]')
 
     // summary table 19444
-    // cy.checkA11y('[data-cy=individualsSummaryTable]')
-    // open action buttons
-    // cy.get('[data-cy=popover-button]').eq(0).click()
-    // cy.checkA11y('[data-cy=individualsSummaryTable]')
+    cy.checkA11y('[data-cy=individualsSummaryTable]', { rules: { 'nested-interactive': { enabled: false } } })
```
Comment (kialj876) [inline btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange.cy.ts:98]: better to move this into its own `it()` test I think

**Review by kialj876 — APPROVED** [review]
Hey Patrick, looks good to me but 1 minor comment before merge

File: btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange.cy.ts (hunk 1 above)
Comment (patrickpeinanw) [reply btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange.cy.ts:98]: Sure. I'll update the test

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-115-235w03tq.web.app

**Pull request merged**